bliss/conf/base_config.yaml

```diff
@@ -227,7 +230,9 @@ surveys:
     dc2:
         _target_: bliss.surveys.dc2.DC2
         data_dir: /nfs/turbo/lsa-regier/lsstdesc-public/dc2/run2.2i-dr6-v4/coadd-t3828-t3829/deepCoadd-results/
-        cat_path: ${paths.dc2}/merged_catalog/merged_catalog_622.pkl
-        batch_size: 128
+        cat_path: ${paths.dc2}/merged_catalog/merged_catalog_psf_100.pkl
```

Comment:
Is the code for creating these pkl files somewhere in our repo, e.g. the scripts directory?

Reply:
If possible, please include it in a case study as part of this PR.